modules/nf-core/sourcepredict/main.nf

```diff
@@ -0,0 +1,58 @@
+process SOURCEPREDICT {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/sourcepredict:0.5.1--pyhdfd78af_0':
+        'biocontainers/sourcepredict:0.5.1--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(kraken_parse)
+    path sources
+    path labels
+    path taxa_sqlite, stageAs:'.etetoolkit/*'
+    path taxa_sqlite_traverse_pkl, stageAs:'.etetoolkit/*'
+
+    output:
+    tuple val(meta), path("*.sourcepredict.csv"), emit: report
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    """
+    export NUMBA_CACHE_DIR='/tmp'
```

Comment:
Writing to `/tmp` can break some clusters, could you write this to `./tmp` or `.` i instead, so a temporary folder is created within the work directory?

Reply:
or if the tmp can be provided via `ext.args` that would also do

Reply:
Changed to ./tmp